src/Illuminate/Support/helpers.php

```diff
@@ -385,7 +385,7 @@ function retry($times, callable $callback, $sleep = 0, $when = null)
     {
         $attempts = 0;
 
-        beginning:
+        beginning :
```

Comment:
We can revert this change. It was introduced by a StyleCI regression.

Reply:
Done